FileSystem.sln

```diff
@@ -11,23 +11,23 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		NuGet.config = NuGet.config
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.FileProviders.Abstractions", "src\Microsoft.Extensions.FileProviders.Abstractions\Microsoft.Extensions.FileProviders.Abstractions.csproj", "{DD94B7E8-3A59-4F84-98A0-8139BE259A87}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.FileProviders.Abstractions", "src\Abstractions\Abstractions.csproj", "{DD94B7E8-3A59-4F84-98A0-8139BE259A87}"
```

Comment:
I mean between different solutions. I agree there's no technical problem in that everything will always build fine, but rather about the filename that would show up in "Recently used files" in various places. Which "Project: Abstractions.csproj" do I click on if there's more than one?